schemas/common/definitions.json

```diff
@@ -198,7 +198,9 @@
   },
   "survey_id": {
     "type": "string",
-    "description": "The survey identifier as used across the ONS.",
+    "description": "The ONS id of the survey derived from the inquiry code already in use for that survey.",
+    "pattern": "^[0-9a-z]+$",
+    "minLength": 1,
```

Comment:
Is the minimum length 1? I would've thought it would be more than that?

Reply:
Yes, Mebin suggested making it `minLength: 1` just to make sure it's not an empty string.

Reply:
Fair enough. Looks good then other than my other comment